spec/controllers/invite_users_controller_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'spec_helper'
+
+describe InviteUsersController do
+  describe "GET new" do
+    before { set_user }
+    it_behaves_like 'require_login' do
+      let(:action) { get :new }
+    end
+
+    it "sets @invite_user variable" do
+      get :new
+      expect(assigns(:invite_user)).to be_a_new(InviteUser)
+    end
+  end
+
+  describe "POST create" do
+    before do
+      set_user
+      ActionMailer::Base.deliveries.clear
+    end
+
+    it_behaves_like 'require_login' do
+      let(:action) { post :create }
+    end
+
+    context "with valid input" do
+      let(:bob) { Fabricate.build(:invite_user) }
+      before { post :create, invite_user: {name: bob.name, email: bob.email, message: bob.message} }
```

Comment:
You can actually use `post :create, invite_user: Fabricate.attributes_for(:invite_user)`, then you don't need the `bob` object at all.